### PR TITLE
Proposed updates nov 2015

### PR DESCRIPTION
**Pull request by jennazee**
Wanted to make some hopefully small updates to the Code of Conduct before we publish it accompanying a blog post soon.

Summary of Changes:
1. Removed the tl;dr because it was not a productive addition to the code (and seemed to violate the spirit of the code itself)
2. Combined the two sections that were entitled "Enactment"
3. Changed "Engineers" to "Individuals" in the Enactment section to make it okay for people other than engineers to call out code of conduct violations
4. Github-flavored Markdown didn't like the previously used superscript method, so I changed it to use the `<sup>` html tag

### CODE REVIEW

**Comment by dje** [comment]
:+1:

**Comment by kitschysynq** [comment]
:100:

**Comment by tspiegs** [comment]
:+1:

**Comment by nickvanw** [comment]
:shipit:

File: code-of-conduct.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
 ## Why have a Code of Conduct?
 This Code of Conduct is designed to help all of us build a pleasant, productive, and fearless community. The purpose of the Code of Conduct is not to burden the team with a bunch of needless rules, or to give us a punishment mechanism for people "being bad," or even to correct things that have been wrong in the past. We are striving to make our engineering team a great group of people to work with, especially for those people who have faced more adverse working environments in the past.
 
-## General
-TL;DR: Don’t be a jerk.
-
-## Social Rules ^1^
+## Social Rules <sup>1</sup>
```
Comment (neurodrone) [inline code-of-conduct.md]: I think we can do something like `<sup>[1](#footnotes)</sup>` here.

**Comment by neurodrone** [comment]
:ship:

File: code-of-conduct.md (hunk 1 above)
Comment (jennazee) [inline code-of-conduct.md]: great idea! thanks!
